manage-data/migrate/migrate-internal-indices.md

```diff
@@ -7,109 +7,35 @@ applies_to:
   deployment:
     eck: unavailable
     ess: ga
-    ece: unavailable
+    ece: ga
   serverless: unavailable
 products:
   - id: cloud-hosted
 ---
 
-# Migrate internal indices
+# Migrate system indices
 
-When you migrate your {{es}} data into a new infrastructure you may also want to migrate your {{es}} internal indices, specifically the `.kibana` index and the `.security` index.
+When you migrate your {{es}} data into a new infrastructure you may also want to migrate your {{es}} system internal indices, specifically the `.kibana` index and the `.security` index.
 
-There are two ways to migrate the internal {{es}} indices:
+In {{es}} 8.0 and later versions, the snapshot and restore of [feature states](/deploy-manage/tools/snapshot-and-restore.md#feature-state) is the only way to back up and restore system indices and system data streams.
```

Comment:
consider making this more positive ("you can use")? maybe with a "this is the only available method" so people don't poke around for alternatives

Reply:
consider also indicating that this method can be _paired_ with another restore method. indicating this makes this page make more sense as a standalone page